hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadata.java

```diff
@@ -131,7 +136,7 @@ static HoodieTableMetadata create(HoodieEngineContext engineContext,
         return metadata;
       }
     }
-
+    LOG.warn("Falling back to FileSystemBackedTableMetadata as metadata table is not initialized");
```

Comment:
minor suggestion. lets log this only when metadataConfig.isEnabled() is true and then we fallback to FS based one